### DISCUSSION TITLE
When will Google really stop support for 3rd party cookies

### DISCUSSION THREAD

**Discussion by Dory998** [privacysandbox/privacy-sandbox-dev-support#368] [General]
the date has moved will this really be in 2025? thanks

**Comment by dmdabbs** [comment]
Yes, that's the $64,000 question.

**Reply by Dory998** [reply]
Talked to a consent management platform vendor today who thinks it will never happen. OneTrust who have huge skin in the game around consent management and control of cookies cant give me an answer as to what their future consent product will look like. whilst some 3rd party vendors around use of products like federated security are actively working on secure API solutions.

**Comment by samdutton** [comment]
> the date has moved will this really be in 2025? 

Hi @Dory998 — as per https://github.com/privacysandbox/privacy-sandbox-dev-support/discussions/370#discussioncomment-9604573, the [1% testing period](https://developers.google.com/privacy-sandbox/blog/cookie-countdown-2024jan) continues through the second half of 2024. Subject to resolving any competition concerns, Chrome plans to begin rolling out third-party cookie phaseout to all Chrome clients starting from early 2025.
[privacysandbox.com/open-web](https://privacysandbox.com/open-web/) shows the timeline.
